lib/rules/no-disallowed-properties.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+module.exports = {
+  'name': 'no-disallowed-properties',
+  'defaults': {
+    'properties': []
+  },
+  'detect': function (ast, parser) {
```

Comment:
Hey @alexeibs ignore the comments from @Snugug with that little codacy icon at the beginning it's some tool that is enabled but doesn't seem to be functioning correctly.. It basically is ignoring the context of all of our work it seems.